### PR TITLE
Automatically set min/max on numeric inputs

### PR DESCRIPTION
**Pull request by jordanmaguire**
I'll be damned if I'm writing those manually.

Will look for a way to unit test this tomorrow

### CODE REVIEW

**Comment by BenR1312** [comment]
is there more coming for this?
Cause i get what its doing, but is it not getting tested?

**Comment by jordanmaguire** [comment]
> is there more coming for this?
Cause i get what its doing, but is it not getting tested?

Yeah - going to look at how simple_form specs its input and copy that. Perhaps on weekend.

**Comment by BenR1312** [comment]
> Yeah - going to look at how simple_form specs its input and copy that. Perhaps on weekend.

so what happens with this one then? wait til those specs or merge away?
Cause it looks good but we cant see exactly what it should be doing?

**Comment by jordanmaguire** [comment]
wait - I wanted to see if this would get any pushback before I invested time in specs.

**Comment by pzi** [comment]
Apart from me not fully understand the codez, you may need to also include the `step` attribute, by default the value of `step` is `1`.

Is this feature going to add the possibility to automagically generate those inputs or are you enhancing simple_form?

**Comment by Ruxton** [comment]
No push back, implement that shit! 👍

**Comment by psyren89** [comment]
Could do with a readme update of how we might use this.

File: app/inputs/numeric_input.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      input_html_options[:max] ||= numericality_validator.options[:max]
+    end
+
+    super
+  end
+
+private
+
+  def numericality_validator
+    object.class.respond_to?(:validators) && object.class.validators_on(attribute_name).find do |validator|
```
Comment (psyren89) [inline app/inputs/numeric_input.rb]: A comment here explaining what's going on would be good.

**Comment by jordanmaguire** [comment]
Made a goof here - will need to compare against the correct fields - greater_than_or_equal_to etc as listed [here](http://guides.rubyonrails.org/active_record_validations.html#numericality)

File: app/inputs/numeric_input.rb (hunk 1 above)
Comment (jordanmaguire) [inline app/inputs/numeric_input.rb]: Good idea

**Comment by jordanmaguire** [comment]
Superseded by https://github.com/thefrontiergroup/frontier_simple_form_numeric_input/pull/1

**Pull request closed without merging**